synapse/storage/databases/main/event_federation.py

```diff
@@ -726,17 +726,39 @@ def _get_auth_chain_difference_txn(
     async def get_backfill_points_in_room(
         self,
         room_id: str,
+        current_depth: int,
+        limit: int,
     ) -> List[Tuple[str, int]]:
         """
         Gets the oldest events(backwards extremities) in the room along with the
```

Comment:
it's not really the oldest any more; rather it is "the oldest with depth lower than `current_depth`"

Reply:
hmmm, it was never really the "oldest" anyway. As we always know about the oldest `m.room.create` event which this doesn't return.

I've updated it to talk about backward extremities instead:

> 
        Get the backward extremities to backfill from in the room along with the
        approximate depth. Only returns events that are at a depth lower than or
        equal to the `current_depth`. Sorted by depth, highest to lowest (descending)
        so the closest events to the `current_depth` are first in the list.

Reply:
that's great, thanks